src/v/cloud_storage/remote.cc

```diff
@@ -130,6 +130,9 @@ ss::future<> remote::stop() {
     co_await _materialized->stop();
     co_await _pool.stop();
     co_await _gate.close();
+    if (ss::this_shard_id() == auth_refresh_shard_id) {
+        co_await _auth_refresh_bg_op.stop();
+    }
```

Comment:
nit: since the shard check when starting is done inside `_auth_refresh_bg_op` it'd be nice if the shutdown check was also in `_auth_refresh_bg_op`. but it's not a big deal.

Reply:
makes sense, fixed